BackendTaskPilot/BackendTaskPilot.csproj

```
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
```

Change the version of .net to .net8 to use the latest one.

```suggestion
        <TargetFramework>net8.0</TargetFramework>
```